# a msg type for motion data from analyze_flow.py
Header header
float32 x_velocity
float32 y_velocity
float32 z_velocity
float32 yaw_velocity
